Esri/arcgis-runtime-toolkit-ios
# Carthage Support

Carthage is the simplest dependency management written by people in Github that supports swift and written in swift. Adding support for Carthage instead of CocoaPods helps developers use frameworks without changes to their projects and creating workspaces. 

Adding support for Carthage will help when getting the lastest of the AR branch @mhdostal 

- Added a travis file, is not required in carthage, just a nice to have for building frameworks. 
- Changed the README file to show how to install the framework
- Exposed the shared scheme needed by carthage

#### Testing
- Add the following to your Cartfile

To test in the fork:
`github "alpascual/arcgis-runtime-toolkit-ios" "Carthage"`

To install a specific version:
`github "esri/arcgis-runtime-toolkit-ios" == 100.5.0`

To install the newest
`github "esri/arcgis-runtime-toolkit-ios"`

To install a minimum release
`github "esri/arcgis-runtime-toolkit-ios" >= 100.4.0`

Make sure carthage it is installed in the mac:
- `brew carthage`
- Then run `carthage update`

#### Resources
https://academy.realm.io/posts/swift-dependency-management-with-carthage/
